faq/buildcuda.inc

```diff
@@ -10,7 +10,27 @@ $a[] = "CUDA-aware support means that the MPI library can send and
 receive GPU buffers directly.  This feature exists in the Open MPI v1.7
 series and later.  The support is being continuously updated so
 different levels of support exist in different versions.  We recommend
-you use the latest v1.8 version for best support.
+you use the latest version for best support.
+
+*Configure and build Open MPI >= 2.0.0 with UCX*
+
+We recommend <a href=\"https://github.com/openucx/ucx/releases/tag/v1.4.0\">UCX1.4</a>
```

Comment:
do we need to specify specific branch here?  this might become older after few days.

Reply:
@bureddy ping

Reply:
👍